Automattic/wp-calypso
# Stats: Refactor the alert icon with the design file

Related to #

## Proposed Changes

* Alter the SVG file with the design document to avoid potential license issues.
* Rename the icon name.

## Testing Instructions


* Follow the testing instructions of https://github.com/Automattic/wp-calypso/pull/84880.
* Ensure the icon for level error on the `NoticeBanner` works as previously.

<img width="1278" alt="截圖 2023-12-11 下午11 16 25" src="https://github.com/Automattic/wp-calypso/assets/6869813/08dd61da-bafd-498e-bc43-44338365d4a2">

## Pre-merge Checklist
